packages/overlay/src/index.tsx

```diff
@@ -22,7 +23,7 @@ export {
   CONTEXT_LINES_ENDPOINT,
   DEFAULT_ANCHOR,
   DEFAULT_EXPERIMENTS,
-  DEFAULT_SIDECAR_URL,
+  DEFAULT_SIDECAR_STREAM_URL as DEFAULT_SIDECAR_URL,
```

Comment:
mistake?

Reply:
no, here we are exporting stream url for users. `DEFAULT_SIDECAR_URL` is an alias here not constant

Reply:
Ah, this is to keep the interface the same. Great!